Disable the profile settings tab. The profile settings tab does not behave in an intuitive way and in most cases is useless. It usually does not work as advertised.
--- a/bsnes/ui-qt/settings/settings.cpp
+++ b/bsnes/ui-qt/settings/settings.cpp
@@ -1,6 +1,6 @@
 #include "../ui-base.hpp"
 
-#include "profile.cpp"
+// #include "profile.cpp"
 #include "video.cpp"
 #include "audio.cpp"
 #include "input.cpp"
@@ -22,17 +22,17 @@
   layout->setSpacing(Style::WidgetSpacing);
   setLayout(layout);
 
-  profileSettingsWindow  = new ProfileSettingsWindow;
+//profileSettingsWindow  = new ProfileSettingsWindow;
   videoSettingsWindow    = new VideoSettingsWindow;
   audioSettingsWindow    = new AudioSettingsWindow;
   inputSettingsWindow    = new InputSettingsWindow;
   pathSettingsWindow     = new PathSettingsWindow;
   advancedSettingsWindow = new AdvancedSettingsWindow;
 
-  profileArea = new QScrollArea;
+/*profileArea = new QScrollArea;
   profileArea->setWidget(profileSettingsWindow);
   profileArea->setFrameStyle(0);
-  profileArea->setWidgetResizable(true);
+  profileArea->setWidgetResizable(true); */
 
   videoArea = new QScrollArea;
   videoArea->setWidget(videoSettingsWindow);
@@ -60,7 +60,7 @@
   advancedArea->setWidgetResizable(true);
 
   tab = new QTabWidget;
-  tab->addTab(profileArea, "Profile");
+//tab->addTab(profileArea, "Profile");
   tab->addTab(videoArea, "Video");
   tab->addTab(audioArea, "Audio");
   tab->addTab(inputArea, "Input");
